src/main/java/com/plaid/client/request/LinkTokenCreateRequest.java

```diff
@@ -0,0 +1,172 @@
+package com.plaid.client.request;
+
+import com.google.gson.annotations.JsonAdapter;
+import com.plaid.client.internal.gson.Optional;
+import com.plaid.client.internal.gson.OptionalDateTypeAdapterFactory;
+import com.plaid.client.request.common.BaseClientRequest;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class LinkTokenCreateRequest extends BaseClientRequest {
+  private User user;
+  private String clientName;
+  private List<String> products;
+  private List<String> countryCodes;
+  private String language;
+  private String webhook;
+  private String accessToken;
+  private String linkCustomizationName;
+  private String redirectUri;
+  private String androidPackageName;
+  private Map<String, SubtypeFilters> accountFilters;
+  private LinkTokenCreateRequest.CrossAppItemAdd crossAppItemAdd;
+  private String paymentInitiation;
+
+  public LinkTokenCreateRequest(
+    User user,
+    String clientName,
+    List<String> products
```

Comment:
Technically, products is not expected for update mode... but it should still work if it's undefined, 🤔

Reply:
I imagine for most use cases this will be specified